src/header/Header.tsx

```diff
@@ -144,6 +146,16 @@ const Header: React.FunctionComponent<HeaderProps> = (props: HeaderProps) => {
             alignItems: 'center',
           },
           backgroundColor && { backgroundColor },
+          elevated && {
```

Comment:
The style needs to be moved to the bottom in the `StyleSheet`

Reply:
Sure Sir